### PR TITLE
Add mark.js

### PR DESCRIPTION
**Pull request by julmot**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/julmot/mark.js
 * Official website (optional, not the repository): https://markjs.io/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 99
   - Count of watchers: 5
   - Count of forks: 13
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

Ping @PeterDaveHello

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Please remove b386655725fcdb09a445465b9408a26814588de2

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help review this PR.

**Comment by julmot** [comment]
@PeterDaveHello When I get to it... I need to clone the repo, as GitHub doesn't offer suchthings. Status after more than one hour: 

`Checking out files:  17% (453509/2627943)`

**Comment by PeterDaveHello** [comment]
You can remove the file online, I'll squash your commit.

**Comment by julmot** [comment]
Great to hear. Just dropped it.

**Comment by PeterDaveHello** [comment]
We need to pass the test first.

**Comment by julmot** [comment]
@PeterDaveHello From my point of view the two complained problems should be ok:

1. The file `mark.min.js` doesn't exist in the repository yes, but on GitHub. I've provided the `autoupdate` section so I expect that files will be added by a bot.
2. The `repository` field is actually valid. Taken from [the docs](https://docs.npmjs.com/files/package.json):
> For GitHub, GitHub gist, Bitbucket, or GitLab repositories you can use the same shortcut syntax you use for npm install:

    ```javascript
"repository": "npm/npm"

"repository": "gist:11081aaa281"

"repository": "bitbucket:example/repo"

"repository": "gitlab:another/repo"
```

**Comment by PeterDaveHello** [comment]
@julmot We do not support that kind of repo syntax yet.

**Comment by julmot** [comment]
@PeterDaveHello 

> We need to pass the test first.

As I said, I'm unable to solve the above named first issue.

**Comment by PeterDaveHello** [comment]
@pvnr0082t @kennynaoh

File: ajax/libs/mark.js/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "repository":{
+    "type":"git",
+    "url":"https://github.com/julmot/mark.js.git"
+  },
+  "autoupdate":{
+    "source":"git",
+    "target":"git://github.com/julmot/mark.js.git",
+    "basePath":"dist",
+    "files":[
+      "*"
```
Comment (pvnr0082t) [inline ajax/libs/mark.js/package.json]: @julmot Let's use `**/*` here.

**Comment by pvnr0082t** [comment]
There is format error of package.json. I suggest that you can use `tools/fixFormat.js` to fix it.

**Comment by pvnr0082t** [comment]
Please also commit the latest version files which in `dist` folder. In detail, you need to create a folder named `6.1.0` under mark.js folder and then put files into the folder.

**Comment by PeterDaveHello** [comment]
> Please also commit the latest version files which in `dist` folder. In detail, you need to create a folder named `6.1.0` under mark.js folder and then put files into the folder.

Another method is to remove the `version` field in package.json, so we'll let the auto-updater to fetch the real files.

**Comment by julmot** [comment]
> There is format error of package.json. I suggest that you can use tools/fixFormat.js to fix it.

I can't clone your repo, it's too large. And since this script requires all `package.json` files from `ajax/libs` I would need to download the entire repo. So, I've done my best to fit the requirements, but it seems to be a bit complicated to contribute here. Could you please do the rest? You have push access to my fork here:
push

**Comment by PeterDaveHello** [comment]
@julmot There is no need to clone the whole repo to get that tool, you can just download it.

**Comment by julmot** [comment]
@PeterDaveHello Since all checks are passing now, is there a need to do any further things?

**Comment by PeterDaveHello** [comment]
@julmot I will take a look at it.

**Pull request merged**